demo-extended/src/main/java/com/novoda/simplechromecustomtabs/demo/ShareBroadcastReceiver.java

```diff
@@ -0,0 +1,22 @@
+package com.novoda.simplechromecustomtabs.demo;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+public class ShareBroadcastReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        String url = intent.getDataString();
+        if (url == null) {
```

Comment:
I don't see you are passing any data on the`Intent` created to trigger this receiver, so I think this will always return. Am I missing something?